uber/site_sections/dept_checklist.py

```diff
@@ -3,32 +3,48 @@
 
 @all_renderable(c.PEOPLE)
 class Root:
-    def index(self, session, message=''):
+    def index(self, session, department_id=None, message=''):
+        if not department_id:
+            raise HTTPRedirect('overview')
+
         attendee = session.admin_attendee()
-        if not attendee.is_single_dept_head:
-            raise HTTPRedirect('overview?message={}', 'The checklist is for department heads with exactly one department')
+        if not attendee.is_dept_head:
+            raise HTTPRedirect('overview?message={}', 'The checklist is for department heads only')
 
+        department_id = Department.to_id(department_id)
+        department = session.query(Department).options(
+            subqueryload(Department.dept_checklist_items)).get(department_id)
```

Comment:
Since these three lines are repeated in three different places, it might be worth eventually making a ``SessionMixin`` utility method so that we can just say

```python
department = session.lookup_dept(department_id)
```

For now it's only occurring three times so that might not be worth it.